lib/node/runner/checker_ballot_transaction.go

```diff
@@ -265,16 +265,15 @@ func ValidateTx(st *storage.LevelDBBackend, tx transaction.Transaction) (err err
 //   source = Account from where the transaction (and ops) come from
 //   tx = Transaction to check
 //
-func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op operation.Operation) (err error) {
+func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op operation.Operation) error {
```

Comment:
IMO, just `return` is fault-prune for this kind of function.
Hence, I make it explicit.